# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/freetype1/gcc33-support.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -urN -x '*~' freetype-1.3.1/test/ftdump.c freetype-1.3.1-gcc-3.3.1/test/ftdump.c
--- freetype-1.3.1/test/ftdump.c	1999-09-04 09:11:29.000000000 +0200
+++ freetype-1.3.1-gcc-3.3.1/test/ftdump.c	2003-08-14 05:13:41.000000000 +0200
@@ -140,7 +140,7 @@
     old_memory += *var;
   }
 
-#define FOOTPRINT( field )  Save_Memory( &memory_footprint.##field )
+#define FOOTPRINT( field )  Save_Memory( &memory_footprint.field )
 
 
   static void
@@ -153,7 +153,7 @@
   }
 
 #define PRINT_MEM( field, string ) \
-          Print_Mem( memory_footprint.##field, string )
+          Print_Mem( memory_footprint.field, string )
 
 
   /* Print the memory footprint */
